test/plugins/run.sh

```diff
@@ -155,16 +155,24 @@ for framework_version in $frameworks; do
   cp -rf ${scenario_home}/* ${case_home}
   cd ${case_home}
 
-  # replace go version
-  sed -i "s/^go [0-9]*\.[0-9]*/go ${go_version}/" go.mod
   # append the module name(for go1.20, module name cannot be same in same workspace)
   mod_case_name=$(echo "${case_name}" | sed 's/\//_/g; s/\./_/g; s/-/_/g')
   mod_name=$(head -n 1 go.mod | cut -d " " -f 2)
-  sed -i "s/^module /module ${mod_case_name}\//" go.mod
-  find . -name "*.go" -type f -exec sed -i "s|${mod_name}|${mod_case_name}/${mod_name}|g" {} \;
-
-  # ajust the plugin replace path
-  sed -i -E '/^replace/ s#(\.\./)#\1../#' go.mod
+  if [ $(uname) = 'Darwin' ]; then
```

Comment:
I would suggest moving conditional check to the top of the file, for example with 'alias' or creating a new function?

Reply:
I think a new function is more reasonable. @ethan256 what do you think?